src/index.js

```diff
@@ -403,8 +402,8 @@ class Masq {
     }
   }
 
-  async _resetLogin () {
-    debug('[masq._resetLogin]')
+  async _resetLogin (keepLoginLink) {
+    debug('[masq._resetLogin] keep login link: ' + keepLoginLink)
```

Comment:
this will display `keep login link:`, maybe you want to explicitly coerce the value to a boolean ?

Reply:
:+1: